src/framework/uicomponents/view/popupview.cpp

```diff
@@ -562,6 +563,27 @@ QRect PopupView::currentScreenGeometry() const
     return currentScreen->availableGeometry();
 }
 
+void PopupView::updateSize()
+{
+    if (!isDialog()) {
+        return;
+    }
+
+    QWindow* qWindow = this->qWindow();
+    if (!qWindow) {
+        return;
+    }
+
+    QSize size = qWindow->size();
+    qWindow->setMinimumSize(size);
+    if (m_resizable) {
+        // These special values (2^24-1) mean "No specific maximum" in Qt
+        qWindow->setMaximumSize(QSize(16777215, 16777215));

```

Comment:
Can we just ignore the setup maximum size as it was before?

Reply:
In practice, yeah, okay. I added it, because I call this method also when the `resizable` property is changed, to make it possible to change that property effectively, even after the dialog has been created. But it is no so likely that we will ever want to do that, so it could be removed.